operator/bundle_helpers/requirements.in

```diff
@@ -0,0 +1 @@
+pyyaml==6.0.2
```

Comment:
Sorry, I'd take back what I suggested before.
If we are to switch to a single `requirements.txt` file after ROX-26860, it could be better put `pytest` here than tediously describe in ROX-26860 all what needs to be done.

I check that adding pytest does not bring any modifications to `requirements-build.txt` so it should be smooth?

Reply:
it is in fact smooth. 
I updated ROX-26860 the remove the gha file once the Python version is bumped.